libs/packages/layouts/src/lib/feature/search-list-layout/search-list-layout.component.ts

```diff
@@ -179,11 +187,37 @@ export class SearchListLayoutComponent implements OnChanges, OnInit {
     this.filterData = filter;
     this.page.pageNumber = this.page.default ? this.page.pageNumber : 1;
     this.page.default = filter ? false : true;
+    this.isDefaultFilter(filter);
     this.updateContent();
```

Comment:
If default filter is true, I think right now we're still making an api call through the updateContent method. Could we add additional condition to not make api call if isDefaultModel is true?

Reply:
Updated the code for it, if it's a default model not making an api call